cpp/generate_new_journal_alert_stats.cc

```diff
@@ -201,7 +200,10 @@ int Main(int argc, char **argv) {
     std::shared_ptr<DbConnection> db_connection(VuFind::GetDbConnection());
     CollectConfigStats(db_connection.get(), user_type, &stats);
     CollectUsageStats(user_type, &stats);
-    GenerateAndMailReport(email_recipient, stats);
+
+    const auto report_file(FileUtil::OpenOutputFileOrDie(REPORT_DIRECTORY + "new_journal_alert_stats." + user_type
```

Comment:
Since this task will run on multiple machines, we need to add the host name to the file name so they will not overwrite each other.

Reply:
Done.